### PR TITLE
[ENG-4387] use osf metrics for node analytics

### PR DESCRIPTION
**Pull request by aaxelb**
-   Ticket: [ENG-4387]
-   Feature flag: n/a

## Purpose
keep the node analytics page running (without keen)

## Summary of Changes
* load data from osf-metrics "node analytics" api
* remove keen js dependencies
* add c3 dependency (for compatibility; used by keen-dataviz)

## QA Notes

[ENG-4387]: https://openscience.atlassian.net/browse/ENG-4387?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by brianjgeiger — APPROVED** [review]
Yay removal of dependencies!

File: lib/analytics-page/addon/application/controller.ts [hunk 1]
```diff
@@ -70,10 +45,10 @@ export default class ApplicationController extends Controller {
     @reads('node.apiMeta.templated_by_count')
     templatedByCount?: number;
 
-    @computed('node.public', 'model.{id,modelName}')
+    @computed('node.public')
     get nodePublic() {
-        const node: Node | null = this.node || this.store.peekRecord(this.model.modelName, this.model.id);
-        return node && node.public;
+        const { node } = this;
+        return !node || node.public;
```
Comment (brianjgeiger) [inline lib/analytics-page/addon/application/controller.ts:51]: I'm not really understanding the `!node` here. Why is node publicness considered true if node is undefined or null?

File: translations/en-us.yml [hunk 2]
```diff
@@ -1753,6 +1752,7 @@ analytics:
         forks: Forks
         registrations: Registrations
         wiki: Wiki
+        metadata: Metadata
```
Comment (brianjgeiger) [inline translations/en-us.yml:1755]: Aren't you missing `resources` here?

**Review by brianjgeiger — CHANGES_REQUESTED** [review]
One question, one thing I think is missing.

File: translations/en-us.yml (hunk 2 above)
Comment (brianjgeiger) [reply translations/en-us.yml:1753]: (And, QA says, also Comments, Components, and Links)

File: lib/analytics-page/addon/application/controller.ts (hunk 1 above)
Comment (aaxelb) [reply lib/analytics-page/addon/application/controller.ts]: good catch, cleaned up

**Pull request merged**